Sources/SwiftProtobufPluginLibrary/Descriptor.swift

```diff
@@ -605,35 +878,59 @@ public final class ServiceDescriptor {
   }
 }
 
+/// Describes an individual service method.
+///
+/// SwiftProtobuf does *not* generate anything for these (or services), but
+/// they are here to support things that generate based off RPCs defined in
+/// .proto file (gRPC, etc.).
 public final class MethodDescriptor {
-  public let proto: Google_Protobuf_MethodDescriptorProto
```

Comment:
Thanks for keeping grpc-swift in mind when making these changes!

grpc-swift uses `proto` for `clientStreaming` and `serverStreaming` which I see are added below. To maintain the API, could these ] properties be added to `MethodDescriptor` in a 1.x release? grpc-swift can bump its minimum required version such that it does not rely on the existence of `proto`.

We could also do the same for `fileDescriptor(named:)` to avoid carrying over `lookupFileDescriptor(named:)` into the 2.0 release.

Reply:
Just put up #1291 to bring those two apis to the 1.x branch.

Also just added a commit to this to keep the needed things but taking them as deprecated, we can consider just dropping them before declaring a 2.0 here if we think there was enough time for things to migrate.

Reply:
ps - There's a little more cleanup I wanted to do, but I'd like to land this first before doing more PRs. Once all that cleanup is done, we could cut a 1.x to get these those changes out, just seems worth waiting to make sure there isn't anything else worth doing to avoid the churn.

Reply:
Great, thank you! We'll adopt them right after the next 1.x release.